File: 1894-merge-strings-alternately/1894-merge-strings-alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        int max_length=Math.max(word1.length(),word2.length());

        int index=0;

        String new_word="";

        while(index<max_length){
            if(index<word1.length()){
               new_word+=word1.charAt(index);
            }
            if(index<word2.length()){
                new_word+=word2.charAt(index);
            }
            index++;
        }
        return new_word;

        

    }
}

File: 1894-merge-strings-alternately/1894-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        max_length=max(len(word1),len(word2))
        index=0
    
    
        new_word=""

        while index<max_length:
            if index<len(word1):
                new_word+=word1[index]
            if index<len(word2):
                new_word+=word2[index]
            index+=1
        return new_word
        
    
        